src/controllers/notifications.js

```diff
@@ -71,6 +73,16 @@ const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 exports.post = function * postNotification () {
   const notification = yield requestUtil.validateBody(this, 'Notification')
   log.debug('Got notification: ' + JSON.stringify(notification))
+
+  const verifySignature = this.config.get('notifications.must_verify')
+  if (verifySignature) {
+    const result = model.verifySignature(notification, this.config)
+    if (!result.valid) {
+      log.warn('Signature verification failed: ' + result.error)
+      throw new UnprocessableEntityError('Notification failed signature verification')
```

Comment:
I can see both arguments:
On one hand, the situation where Ledger's sending an invalid signature is so bad (certificate expired, private key misconfigured, etc.) that maybe it's fine to fail dramatically. 
On the other hand, unfortunately, these things tend to happen in real life.  When it happens, we wouldn't want to flood the servers. 
How about logging an error and returning 200, like we do here?
https://github.com/interledger/five-bells-connector/blob/master/src/controllers/notifications.js#L81